packages/block-editor/src/store/selectors.js

```diff
@@ -1421,19 +1383,16 @@ export function __unstableIsLastBlockChangeIgnored( state ) {
 }
 
 /**
- * Returns the value of a post meta from the editor settings.
+ * Returns the block attributes changed as a result of the last dispatched
+ * action.
  *
- * @param {Object} state Global application state.
- * @param {string} key   Meta Key to retrieve
+ * @param {Object} state Block editor state.
  *
- * @return {*} Meta value
+ * @return {Object<string,Object>} Subsets of block attributes changed, keyed
+ *                                 by block client ID.
  */
-function getPostMeta( state, key ) {
-	if ( key === undefined ) {
-		return get( state, [ 'settings', '__experimentalMetaSource', 'value' ], EMPTY_OBJECT );
-	}
-
-	return get( state, [ 'settings', '__experimentalMetaSource', 'value', key ] );
+export function getLastBlockAttributesChange( state ) {
```

Comment:
How confident are we about the long term validity of this selector? should it be experimental for some time? (I guess same question for the two actions being introduced)

Reply:
> How confident are we about the long term validity of this selector? should it be experimental for some time? (I guess same question for the two actions being introduced)

I think it would always almost always for internal use, but I suppose that doesn't help us much as far as backwards-compatibility commitment. I don't mind to mark them as experimental for a few releases.

I don't know if we might consider another sort of unstable naming convention for `__internal` methods, which this would fit. There is some precedent with WordPress and its underscore-prefixed functions (e.g. [`_wp_admin_bar_init`](https://developer.wordpress.org/reference/functions/_wp_admin_bar_init/)).

Reply:
> How confident are we about the long term validity of this selector? should it be experimental for some time? (I guess same question for the two actions being introduced)

I'm going to change these to be experimental prefixed.

As an aside, I also questioned how this should be named considering pluralization, since there can be one or multiple blocks which have one or multiple attributes changed. Having it as experimental would let us rename if we settle on a better form (`getLastBlocksAttributesChanges`? `getLastBlocksAttributeChanges`? `getLastBlockAttributeChanges`?)

Reply:
>`getLastBlockAttributeChanges`

It returns changes to attributes.

Reply:
> > `getLastBlockAttributeChanges`
> 
> It returns changes to attributes.

Works for me. Updated in d955e89.